rest-server/src/routes/user.js

```diff
@@ -31,5 +31,8 @@ router.route('/')
     /** DELETE /api/v1/user - Remove a user */
     .delete(tokenConfig.check, param.validate(userConfig.userDeleteInputSchema), userController.remove);
 
+router.route('/:username/virtualClusters')
+    .put(tokenConfig.check, param.validate(userConfig.userVcUpdateInputSchema), userController.updateUserVc);
```

Comment:
There is no 'GET' API?

Reply:
we only add access control part, not support 'GET'

Reply:
Only add access control part, we don't support 'GET' API now.